tests/integration/ha_tests/test_smoke.py

```diff
@@ -107,6 +106,8 @@ async def test_charm_garbage_ignorance(ops_test: OpsTest, charm: str):
         for attempt in Retrying(stop=stop_after_delay(30 * 3), wait=wait_fixed(3), reraise=True):
             with attempt:
                 garbage_storages = await get_detached_storages(ops_test)
+                assert len(garbage_storages) == 4
+                logger.info(f"Collected storages: {garbage_storages}")
```

Comment:
Before we waited for a single storage to detach, but now we need all four of them to redeploy.

Reply:
Good check.